docs/core/resilience/http-resilience.md

```diff
@@ -3,7 +3,7 @@ title: "Build resilient HTTP apps: Key development patterns"
 description: Learn how to build resilient HTTP apps using the Microsoft.Extensions.Http.Resilience NuGet package.
 author: IEvangelist
 ms.author: dapine
-ms.date: 10/20/2023
+ms.date: 07/01/2024
 ---
 
 # Build resilient HTTP apps: Key development patterns
```

Comment:
Shall we also include what status codes and exceptions are handled for retries and circuit breaker?

https://github.com/dotnet/extensions/blob/afde87b8812ef0b1dcee5db35cb3a49ca6dc9197/src/Libraries/Microsoft.Extensions.Http.Resilience/Polly/HttpClientResiliencePredicates.cs#L25